test/e2e/common/node/lifecycle_hook.go

```diff
@@ -629,3 +629,36 @@ var _ = SIGDescribe(feature.PodLifecycleSleepAction, func() {
 
 	})
 })
+
+var _ = SIGDescribe(feature.PodLifecycleSleepActionAllowZero, func() {
```

Comment:
were there any tests to disable feature gate in scope of a KEP?

Reply:
> a test with feature.PodLifecycleSleepActionAllowZero disabled, for example, when fg is disabled, creating a pod with 0 duration will result in a error?

Ah yes, I have added a test case in `validation_test.go` for this:
https://github.com/kubernetes/kubernetes/blob/a0d4262276ee5db42e9104f7640d6d800c9e73d3/pkg/apis/core/validation/validation_test.go#L24180-L24188

Reply:
ok, the feature is simple enough to not have an e2e_node test for it.